### DISCUSSION TITLE
Can I run Oracle Database containers on Apple M1 (Arm) devices?

### DISCUSSION THREAD

**Discussion by doberkofler** [oracle/docker-images#2669] [Q&A]
It has been a long wait, but now the docker-images [FAQ](https://github.com/oracle/docker-images/blob/main/OracleDatabase/SingleInstance/FAQ.md) now suggests that [Oracle 19c for ARM](https://www.oracle.com/database/technologies/oracle19c-linux-arm64-downloads.html) can be installed natively on Apple Silicon.

![image](https://github.com/oracle/docker-images/assets/1009585/a599cf19-3088-4b93-a4ae-87ac43afc6ad)

As far as I understand  it should be possible run the following stack:
Oracle 19c for ARM
Oracle Linux for ARM
Docker Desktop for Apple Silicon
Apple Silicon based hardware

Did someone already succeed in natively installing an Oracle 19c  ARM Database on Apple Silicon?
What exact component have been used?
How is the performance?

Thank you!

**Comment by oehrlis** [comment]
Hi

I could sucessfully setup and run a ARM based Docker Container with Oracle 19c ARM and Oracle Linux ARM in MacOS with an M1 and M2.

io calibrate did Show much better Performance. Blog Post is still pending

regards
Stefan

**Comment by oraclesean** [comment]
I'm running it through Docker, too: https://github.com/oraclesean/cloud-native-oracle. The "eyeball benchmark" performance of a 2021 MBP M1 with 16G RAM far exceeds that of a 2018 MBP 2.2GHz i7 with 32G RAM. Building an image takes under 5 minutes; creating a database takes about 10. (It's also faster than a 4-CPU VM.Standard.E4.Flex instance on OCI.)

**Comment by simonpcouch** [comment]
For folks who saw that FAQ entry screenshotted above and weren't sure how to get started, I [just put up a blog post](https://www.simonpcouch.com/blog/2024-03-14-oracle/) that walks through that process step-by-step.🙂
